certbot/certbot/_internal/cli/helpful.py

```diff
@@ -223,6 +223,14 @@ def parse_args(self) -> argparse.Namespace:
                 raise errors.Error("Using --allow-subset-of-names with a"
                                    " wildcard domain is not supported.")
 
+        # error when a private key already exists, and it's not a new account
+        if parsed_args.ecdsa_account_key:
```

Comment:
I'm not sure what you mean.

This variable seems to be set to True or False depending on whether or not `--ecdsa-account-key` is set which seems like what you want. Maybe you have some tooling that's complaining about the flag not existing because [`NamespaceConfig` just proxies the request through to the `argparse.Namespace` object](https://github.com/certbot/certbot/blob/142fcad28b0e61f335b92a743b38c8d677bd00dd/certbot/certbot/configuration.py#L55-L58) and the tooling doesn't understand that?